### DISCUSSION TITLE
router.isFallback problem with Facebook share

### DISCUSSION THREAD

**Discussion by IRediTOTO** [vercel/next.js#20574] [Help]
# Feature request
Make `getStaticPaths` have `fallBack===true` work like `getServerSideProps `on first time load

## Is your feature request related to a problem? Please describe.
I have a single product page with `fallback: true,` When I post new product it can be loaded on.
But problem is when I post new product, it will share it auto on Facebook page, post image in SEO tag didn't exist yet so it take image from homepage => No image of product show on shared post on Facebook. I know because this code
```
if (router.isFallback) {
      return (
         <Layout>
            <LoadingAnimate />
         </Layout>
      );
   }
```
No data to show with Facebook on first time load. 

## Describe the solution you'd like

Make an option to getStaticPaths work like serverSideProps. Wait for page render then send to client, not send immidiate.

**Comment by timneutkens** [comment]
This is already supported with `fallback: 'blocking'`.

**Reply by IRediTOTO** [reply]
I got it. Didnt see it in docs
